src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import org.junit.Test;
+
+public class WriterServiceImplTest {
+    private final WriterService writerService = new WriterServiceImpl();
```

Comment:
lets add tests for null and  invalid file names

Reply:
Any filename is up to us so there's no such thing as an invalid name. Maybe we want to create a few with different names in the future. The main point filename must exist so checking for null is understandable.